src/Workspaces/Core/Portable/Workspace/Solution/ITextVersionable.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal interface ITextVersionable
+    {
+        bool TryGetTextVersion(out VersionStamp version);
```

Comment:
Given there is only one implementation and it always returns true should we just make this GetTextVersion?